load/LoadOrganizations.py

```diff
@@ -145,8 +149,10 @@ def process_other_agreement(self, language_record, type_code, fileset_id):
 					licensors.append(language_record.Licensor())
 				else:
 					print(f"ERROR Agreement Type is 'Other' and Methodology 'Joint' but licensor is empty for fileset id: {fileset_id}")
+			elif type_code == "audio":
+				licensors.append(self.LicensorHosanna)
 			else:
-				print(f"ERROR Agreement Type is 'Other', Methodology is 'Joint', but type is not text. Unable to assign for fileset id: {fileset_id}")
+				print(f"ERROR Agreement Type is 'Other', Methodology is 'Joint', but type is not text or audio. Unable to assign for fileset id: {fileset_id}")
```

Comment:
same: we'll handle video separately, so remove this

Reply:
This log has been removed as you can see here: https://github.com/faithcomesbyhearing/dbp-etl/pull/136/files#diff-0628aedffa821d1b9acf4c81d50c756fe72674fcc7ca34c47e0a5b0a65c60aceL149